### DISCUSSION TITLE
Is there a parameter to ignore tls verification on read of source ?

### DISCUSSION THREAD

**Discussion by chddv** [bluenviron/mediamtx#3762] [Questions] [answered]
### Question

Hi,
I have 2 mediamtx runing in docker.
The first one produce a WebRTC stream. I can open it in Chrome without problem. 
My second mediamtx must read this source. It did not work because my stream is in https, with a invalid self signed certificate. 
_"tls: failed to verify certificate: x509: certificate signed by unknown authority"_

Is there an option to ignore TLS verification in source ?
Thanks.

**Comment by aler9** [comment, marked as answer]
yes, the option is called `sourceFingerprint` and is documented in the configuration file:
https://github.com/bluenviron/mediamtx/blob/f9162b230348ab237d3d7d4b4999c469ce46fb9a/mediamtx.yml#L428-L433

**Comment by chddv** [comment]
Hi 
Thanks for your answer ! 
Christophe

[a comment by an automated account is not shown]
